rust/template/differential_datalog/src/ddval/ddval_convert.rs

```diff
@@ -149,7 +149,8 @@ where
             <*const usize>::cast::<Self>(&value.v).read()
         } else {
             let arc = Arc::from_raw(value.v as *const Self);
-            Arc::try_unwrap(arc).unwrap_or_else(|a| (*a).clone())
+            // FIXME: An equivalent of `Arc::try_unwrap()` for triomphe
+            Self::clone(&*arc)
```

Comment:
Isn't this terrible for performance: we're cloning every time even when we don't need to (which I expect to be the common case)?